superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FiltersConfigForm/utils.ts

```diff
@@ -20,9 +20,15 @@ import { flatMapDeep } from 'lodash';
 import { FormInstance } from 'antd/lib/form';
 import React from 'react';
 import { CustomControlItem, DatasourceMeta } from '@superset-ui/chart-controls';
+import { Column, ensureIsArray, GenericDataType } from '@superset-ui/core';
 
 const FILTERS_FIELD_NAME = 'filters';
 
+export const FILTER_GROUPS = {
+  TIME: ['filter_time', 'filter_timegrain', 'filter_timecolumn'],
```

Comment:
nit: would `TEMPORAL` be more accurate? Also, would it be a good idea to list the supported datatypes instead?

```js
{
  filter_time: [GenericDataType.TEMPORAL],
  filter_timegrain: [GenericDataType.TEMPORAL],
  filter_timecolumn: [GenericDataType.TEMPORAL],
  filter_select: [GenericDataType.STRING, GenericDataType.NUMERIC, GenericDataType.TEMPORAL],
  filter_range: [GenericDataType.NUMERIC],
}
```
or similar

Reply:
Fantastic suggestion, thank you!